arb/bin.c

```diff
@@ -10,42 +11,106 @@
 */
 
 #include "arb.h"
+#include "flint/fmpz.h"
+
+#define LOG2_BIN(n, k)                      \
+    (-1.3257480647361592                    \
+     + ((n) + .5) * log2(n)                 \
```

Comment:
I think log2 might not be available everywhere. Better to use log.

Reply:
What is your opinion about supporting/assuming newer standards?